app/grants/migrations/0118_auto_20210318_2150.py

```diff
@@ -0,0 +1,50 @@
+# Generated by Django 2.2.4 on 2021-03-18 21:50
+
+import logging
+from django.db import migrations
+
+logger = logging.getLogger(__name__)
+
+
+def forwards(apps, schema_editor):
+    CLRMatch = apps.get_model('grants', 'CLRMatch')
+    GrantCLR = apps.get_model('grants', 'GrantCLR')
+
+    # round_number to pk
+    mapping = {
+        1: 7,
+        2: 8,
+        3: 9,
+        4: 10,
+        5: 11,
+        6: 12,
+        7: 2,
+        8: 18,
+        107: 1,
+        108: 14,
+        110: 110,
+        120: 20,
+        121: 21,
+    }
+
+    for key, val in mapping.items():
+        gclr = GrantCLR.objects.get(pk=val)
```

Comment:
Should we wrap this in a try catch? I don't think my local is set-up with these GrantCLRs and the migration is failing

Reply:
done